src/codegen/object.rs

```diff
@@ -607,4 +608,14 @@ pub fn generate_reexports(
             module_name, analysis.trait_name
         ));
     }
+
+    if has_builder_properties(&analysis.builder_properties) {
+        for cfg in &cfgs {
+            builders.push(format!("\t{}", cfg));
+        }
+        builders.push(format!(
+            "\t pub use super::{}::{}Builder;",
```

Comment:
Please don't use `\t` and write four whitespace directly. In this case it's even more weird considering that there is a whitespace just after...

Reply:
I copied whatever used few lines above

Reply:
oh never mind; i didn't intend to add the whitespace after \t